BUILDING.md

```diff
@@ -317,37 +317,39 @@ To run the linter without running tests, use
 If you are updating tests and want to run tests in a single test file
 (e.g. `test/parallel/test-stream2-transform.js`):
 
-```text
-$ tools/test.py test/parallel/test-stream2-transform.js
-```
+````text
+$ python3 tools/test.py test/parallel/test-stream2-transform.js
```

Comment:
This is necessary only if you don't have a `python` executable on your path, I don't know how common that is.
````suggestion
```text
$ python3 tools/test.py test/parallel/test-stream2-transform.js
```
````

Reply:
Nice catch 🙌 

It is probably why the formatter added the fourth backtick.

> This is necessary only if you don't have a python executable on your path, I don't know how common that is.

I put a note, let me know WDYT.

Reply:
I wonder if the executable for Python 3 will be renamed from `python3`  to `python` now that Python 2 is dead, in which case this change would need to be reverted. I guess that's a problem for later, let's not overthink this.